build-tools/xaprepare/xaprepare/Steps/Step_InstallDotNetSDKs.cs

```diff
@@ -0,0 +1,113 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace Xamarin.Android.Prepare
+{
+	class Step_InstallDotNetSDKs : StepWithDownloadProgress
+	{
+		public Step_InstallDotNetSDKs ()
+			: base ("Install required .NET Core SDKs locally")
+		{ }
+
+		protected override async Task<bool> Execute (Context context)
+		{
+			if (context == null)
```

Comment:
This check shouldn't be necessary anymore since `xaprepare` is compiled with nullable refs and thus `context` can be assumed not to be `null` here.